### DISCUSSION TITLE
https://thomashallam.github.io/posts/2022-11-18-weeknotes/nhsd-ur-lab-launch.html

### DISCUSSION THREAD

**Discussion by giscus** [thomashallam/thomashallam.github.io#7] [General]
# https://thomashallam.github.io/posts/2022-11-18-weeknotes/nhsd-ur-lab-launch.html

https://thomashallam.github.io/posts/2022-11-18-weeknotes/nhsd-ur-lab-launch.html

**Comment by thomashallam** [comment]
CHATGPT:

Based on the quotes provided, I see several themes emerging:

Successful launch event: The quotes express positive feedback about the launch event, which was well-attended by colleagues from across the business. The event was organized by Rochelle, Allan, and Ellen and was highlighted as great work. The CEO and Director of Product Development were present and enthused about the lab and the technology.

Reflecting on a long journey: The quotes mention that the launch event provided an opportunity to reflect on the long journey of building the UR lab during the pandemic. The person providing the quotes mentions that they have been involved in the lab design journey since 2018/2019, playing a key role in facilitating the discovery of requirements with the user research community, understanding the art of the possible for the lab, designing the end-to-end user experience and working closely with suppliers and internal teams.

Positive feedback on the technology: The quotes express that all the technology stood up and people had fun playing with the PTZ cameras and the event was streamed for colleagues with MS Teams. The lab was recorded to capture any feedback. There is also excitement about several teams having projects coming up where they may be ready to use the venue and interest from other health departments.

Upcoming challenges and puzzles: The quotes mention that the UR lab really needs a 'duty manager' to take it forward and running this lab would be a great development opportunity for whoever takes on that kind of challenge. The person providing the quotes also mentions that they are a starter and not often a finisher and they are sad that 99% of the UR lab challenges have now been resolved.

In-person workshops: The quotes mention that the person providing them is travelling to London in a few weeks for a big workshop, but they are not looking forward to the long train rides. They ask for suggestions for entertainment on the way.

Appreciation for thoughts and feedbacks: The person providing the quotes also express appreciation for any thoughts and feedback massively appreciated.

It's important to note that this is one person's perspective and there may be other themes or insights that would emerge from analyzing feedback from a diverse group of people. Additionally, it would be beneficial to prioritize the themes based on the business context and the goals of the UR lab.
